### ISSUE TITLE
Icons missing in Windows high contrast model for FF

### ISSUE THREAD

**Issue by nicoxxie** [IBM/carbon-icons#98]
Received an a11y issue on the icon does not work in Firefox high contrast theme.  I verified this works on Chrome but not FF.

![screen shot 2019-01-28 at 12 10 06 pm](https://user-images.githubusercontent.com/17915105/52575088-f265b280-2e60-11e9-879b-3b45661fe310.png)

https://github.ibm.com/IAM/access-management/issues/8027
